### DISCUSSION TITLE
What does "RN" mean?

### DISCUSSION THREAD

**Discussion by MesMar12** [egzumer/uv-k5-firmware-custom#593] [Q&A] [answered]
Hi, I just didn't find what RN means displayed at the bottom of the screen when I'm on band 7.

**Comment by prokrypt** [comment, marked as answer]
Just guessing:
R = reverse (F+8)
N = narrow bandwidth

**Reply by MesMar12** [reply]
Yes, it's that. The letters are a bit close together, so I read it as one symbol.
